trino/integration_test.go

```diff
@@ -751,6 +760,28 @@ func TestIntegrationQueryContextCancellation(t *testing.T) {
 	}
 }
 
+func TestIntegrationBearerAuth(t *testing.T) {
+	bearerAuth := "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ0ZXN0IiwiZXhwIjo0ODY3MjQ4MzY1fQ.a8O5YhYSn1WNedLqAYq8e5lXSwOkabZjhvKIAXPEGBc5juYBgnWcfad4kXn4PCoD6ykcmGACRuMiBNGbQMxPy6iX5vNie8tX-P-GuXhgHuRIcf6deVOibGb9RHaS0VwtwsgBZREhxF9-410db0k5qwipVJKu-t8mttvgIRhx2hyRaXigH2IT1W3Q06K3o81Olq-c_b4qF1oVpVEuQ7uzPUXb5LVK30YVBcUtFHtlYQX0ZJ_5hCTI37l5krBGIt--WOGs368tVrUPDLyhdEW2V7dN_NnPwfGuxzBVRTgC8oqGNlbOWSouUYrfW3gN6LYEckBfXGbk5daNQ5u0OmYkgg"
```

Comment:
Is this token different from `trino/etc/secrets/jwt_token`? Do we need both? How long it's valid for?

Reply:
Hi @nineinchnick - Thanks for your feedback. 
The JWT token has a validity of 100 years and I can remove all the files in the directory that are not used (just added for completeness). Also, I will add steps in Readme file in case we need to generate the secrets again.

Reply:
Yes, please remove unused files.

Reply:
@nineinchnick - Removed unused files